### DISCUSSION TITLE
[REGRESSION] Cannot create tags with spaces

### DISCUSSION THREAD

**Discussion by KyleSanderson** [jesec/flood#300] [Ideas]
Type: Bug Report

- [x] Try to follow the update procedure described in the README and try again before opening this issue.

## Your Environment

- Version used:
  - ACK
- Environment name and version:
  - Web browser `Chrome 90`
- Operating system and version: Windows 10 LTSC N
- Torrent client and version: Flood regression

## Summary

rtorrent supports tags with spaces, jesec's flood does not.

## Expected Behavior

Spaces should be allowed again as per normal application usage.
## Current Behavior

Spaces are not allowed.

## Possible Solution

Revert the commit(?).
## Steps to Reproduce

Add a tag, spaces are denied.

1.
2.
3.
4.

## Context

Add a basic tag.

**Comment by jesec** [comment]
Spaces are never allowed. This is not a regression. 

Moving to discussions as this is a feature request.
